### ISSUE TITLE
KeyPrefix not working when SyncUpload a directory

### ISSUE THREAD

**Issue by swandive0209** [SharpeRAD/Cake.AWS.S3#6]
When using SyncUpload method to sync a local directory, no matter what KeyPrefix I specified. The local folder will always be put in the bucket root folder on S3. 

The GetKey method in S3Manager.cs seems not using the keyPrefix, it just use the local folder path as the S3 key.

```
string key = this.GetKey(file, fullPath, settings.LowerPaths);
```

**Comment by SharpeRAD** [comment]
So fork the repository and submit a PR...

**Comment by SharpeRAD** [comment]
Had it open when you messaged so it'll be fixed in v0.3.1 😄
